frontend/include/chpl/framework/stringify-functions.h

```diff
@@ -301,6 +301,14 @@ void operator()(std::ostream& streamOut,
 }
 };
 
+template<> struct stringify<char> {
+void operator()(std::ostream& streamOut,
+                StringifyKind stringKind,
+                const char val) const {
+  streamOut << std::to_string(val);
```

Comment:
According to my in-depth research ([a single stack overflow post](https://stackoverflow.com/questions/46898203/c-converting-char-variable-to-string-variable-is-returning-ascii-value)), doesn't this print the int value of the char? Does
```C++
streamOut << val;
```
not work for our purposes?